docs/pages/includes/kubernetes-access/confirm-k8s.mdx

```diff
@@ -0,0 +1,63 @@
+<Details
+title="Confirm Kubernetes is enabled in Teleport"
+scope={["oss", "enterprise"]}
+scopeOnly={true}
+opened={true}
+>
+
+Confirm the `Kubernetes` status is `enabled` within the cluster and the user has Kubernetes groups
```

Comment:
I think this could use a short introductory sentence/phrase for why we're confirming this information. 

For example, "Teleport Kubernetes Access must be enabled before you can use this guide. Confirm that..."